lib/PanelTestProvider.jsm

```diff
@@ -79,9 +79,9 @@ const MESSAGES = () => [
     // Never saw this message or saw it in the past 4 days or more recent
     targeting: `isWhatsNewPanelEnabled &&
       (earliestFirefoxVersion && firefoxVersion > earliestFirefoxVersion) &&
-        messageImpressions[.id == 'WHATS_NEW_BADGE_${FIREFOX_VERSION}']|length == 0 ||
-      (messageImpressions[.id == 'WHATS_NEW_BADGE_${FIREFOX_VERSION}']|length >= 1 &&
-        currentDate|date - messageImpressions[.id == 'WHATS_NEW_BADGE_${FIREFOX_VERSION}'][0] <= 4 * 24 * 3600 * 1000)`,
+        (messageImpressions['WHATS_NEW_BADGE_${FIREFOX_VERSION}'] ||
```

Comment:
don't you mean `!messageImpressions['WHATS_NEW_BADGE_${FIREFOX_VERSION}']`

Reply:
yes